container-storage-setup.1

```diff
@@ -88,8 +88,13 @@ functionality to work properly.
 STORAGE_DRIVER:
       Specify a storage driver to be used with container runtime.
       Default: "devicemapper".
-      Valid values are overlay, overlay2 and "".
-      "" tells container-storage-setup to not perform any storage setup.
+      Valid values are devicemapper, overlay, auto, and "".  The empty string ""
+      is used when you want to ensure container-storage-setup does not automatically
+      change configuration (for example, you want to do it manually).  The "auto" option
+      may be used when c-s-s is started by default (for example, Atomic Host);
```

Comment:
Can we specify that `auto` option means that css will determine storage driver automatically. choosing overlay2 first  is only current behavior and it can change down the line. Don't want users to rely on this.